tests/libres_tests/res/enkf/test_runpath_list_ert.py

```diff
@@ -18,7 +18,8 @@ def test_assert_symlink_deleted(setup_case):
     # replace field file with symlink
     linkpath = f"{run_context[0].runpath}/permx.grdcel"
     targetpath = f"{run_context[0].runpath}/permx.grdcel.target"
-    open(targetpath, "a").close()
+    with open(targetpath, "a"):
+        pass
```

Comment:
Do you know why we open a file and just `pass`?

Reply:
It is equivalent to the bash `touch` command, which either makes an empty file if it does not exist, or if it exists, updates the time of last access.